src/persist/src/operators/input.rs

```diff
@@ -136,11 +137,13 @@ mod tests {
             p.into_inner()
         });
 
-        let mut p = MemPersister::from_inner(dataz);
+        // TODO: I don't get what this second scope is meant to illustrate?
```

Comment:
A restarted process! I had a comment here to that effect at some point but looks like I lost it along the way